api/src/test/java/org/openmrs/module/bedmanagement/BedManagementServiceImplTest.java

```diff
@@ -34,6 +35,7 @@
 import static org.mockito.Mockito.when;
 import static org.powermock.api.mockito.PowerMockito.mockStatic;
 
+@PowerMockIgnore("javax.management.*")
```

Comment:
Tests were failing without this, which is pretty common.  Not sure how/if this was working previously, but it is likely environmental.